dcs/src/main/java/org/trafodion/dcs/servermt/serverHandler/ServerApiGetCatalogs.java

```diff
@@ -353,7 +353,7 @@ else if (columnType == ServerConstants.SQL_ROWVER){
                     case ServerConstants.SQL_API_SQLSTATISTICS:
                         if(LOG.isDebugEnabled())
                             LOG.debug(serverWorkerName + ". getIndexInfo (catalogNm :" + catalogNm + ", schemaNm :" + schemaNm + ", tableNm :" + tableNm + ", uniqueness :" + ((uniqueness == 1)? true : false) + ", true)");
-                        rs = dbmd.getIndexInfo(catalogNm, schemaNm, tableNm, (uniqueness == 1)? true : false, true);
+                        rs = dbmd.getIndexInfo(catalogNm, schemaNm, tableNm, (uniqueness == ServerConstants.SQL_INDEX_ALL)? true : false, true);
```

Comment:
Thanks for the updates. Here shouldn't this be SQL_INDEX_UNIQUE (0 - like your previous change). You could update line 355 also.